src/index.ts

```diff
@@ -127,7 +127,7 @@ try {
 		const project = await getProject();
 		if (!project) throw new Error("Unable to find pages project");
 
-		const productionEnvironment = githubBranch === project.production_branch;
+		const productionEnvironment = [githubBranch, branch].includes(project.production_branch);
```

Comment:
Not a fan of doing includes here. If my prod branch is `main` (the Git default) and I make a branch `walshy/maintenance` it'll now be a "prod" deploy.

Reply:
My intention here was to have an array with branch names, and see if the production branch is included in the array. It shouldn't be about checking if a single string includes the other branch name...

For example:
```ts
['main', 'other-branch'].includes('main') // true because 'main' is in the array
```

Reply:
But I'm happy to change it into two `conditional || conditional` too 👍

Reply:
oh wait yeah this is me being dumb in the morning, this is _array_ includes not a _string_ includes 🤦